docs/content/deprecated/ActionList.mdx

```diff
@@ -0,0 +1,136 @@
+---
+componentId: action_list
+title: ActionList
+status: Deprecated
+source: https://github.com/primer/react/tree/main/src/deprecated/ActionList
+---
+
+An `ActionList` is a list of items which can be activated or selected. `ActionList` is the base component for many of our menu-type components, including `DropdownMenu` and `ActionMenu`.
+
+## Deprecation
+
+Use [composable API instead](/ActionList) instead.
```

Comment:
Shall we say this has been superseded with a new version instead? It has a lot of other qualities besides composable API and will receive future support, which is the main reason for them to migrate.

Reply:
Changed the copy here.